### PR TITLE
PMM-7 Fix pmm3-aws-staging-start pipeline

### PR DESCRIPTION
**Pull request by ademidoff**

### CODE REVIEW

File: vars/setupPMMClient.groovy [hunk 1]
```diff
@@ -98,12 +96,8 @@ def call(String SERVER_IP, String CLIENT_VERSION, String PMM_VERSION, String ENA
 
             pmm-admin --version
             if [[ "$CLIENT_VERSION" =~ dev-latest|pmm2-latest|pmm2-rc|^2.* ]]; then
-                if [[ "$CLIENT_INSTANCE" = yes ]]; then
```
Comment (ademidoff) [inline vars/setupPMMClient.groovy:101]: Same stuff but less code :)

File: vars/setupPMMClient.groovy [hunk 2]
```diff
@@ -47,10 +47,8 @@ def call(String SERVER_IP, String CLIENT_VERSION, String PMM_VERSION, String ENA
                 sudo yum -y install "pmm2-client-$CLIENT_VERSION-6.el7.x86_64"
                 if [[ "$ENABLE_TESTING_REPO" = yes ]]; then
                     sudo percona-release enable-only original testing
-                elif [[ "$ENABLE_TESTING_REPO" = no ]]; then
+                elif [[ "$ENABLE_TESTING_REPO" = no ]] && [[ "$ENABLE_EXPERIMENTAL_REPO" = yes ]]; then
                     sudo percona-release enable-only original experimental
-                elif [[ "$ENABLE_TESTING_REPO" = no ]] && [[ "$ENABLE_EXPERIMENTAL_REPO" = no ]]; then
```
Comment (ademidoff) [inline vars/setupPMMClient.groovy:52]: Same stuff, less code

File: pmm/v3/pmm3-aws-staging-start.groovy [hunk 3]
```diff
@@ -256,11 +260,6 @@ pipeline {
                         popd
                     '''
                 }
-                script {
```
Comment (ademidoff) [inline pmm/v3/pmm3-aws-staging-start.groovy:259]: This was redundant

File: IaC/pmm.cd/init.groovy.d/matrix.groovy [hunk 4]
```diff
@@ -144,7 +144,6 @@ authz_strategy_config = [
     strategy: 'GlobalMatrixAuthorizationStrategy',
     user_permissions: [
         'percona*build-engineers': ['Overall Administer'],
-        'percona*iit': ['Overall Administer'],
```
Comment (ademidoff) [inline IaC/pmm.cd/init.groovy.d/matrix.groovy:147]: Remove a non-existent group

File: pmm/v3/pmm3-aws-staging-start.groovy [hunk 5]
```diff
@@ -8,7 +8,10 @@ library changelog: false, identifier: 'lib@master', retriever: modernSCM([
     remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
 ]) _
 
-def DEFAULT_SSH_KEYS = getSHHKeysPMM()
+library changelog: false, identifier: 'v3lib@PMM-7-fix-pmm3-aws-staging-start-ppl', retriever: modernSCM(
```
Comment (ademidoff) [inline pmm/v3/pmm3-aws-staging-start.groovy:11]: TODO: change branch to `master` before merging

**Review by vorsel — APPROVED** [review]

File: IaC/pmm.cd/init.groovy.d/matrix.groovy (hunk 4 above)
Comment (ademidoff) [reply IaC/pmm.cd/init.groovy.d/matrix.groovy]: Reverted )

**Pull request merged**